README.md

```diff
@@ -386,7 +412,7 @@ A Promise that resolves with the response.
 
 #### Description
 
-This method is used to send http requests to the Identity Server. The developer doesn't need to manually attach the access token since this method does it automatically.
+This method is used to send http requests to the Asgardeo. The developer doesn't need to manually attach the access token since this method does it automatically.
```

Comment:
```suggestion
This method is used to send http requests to the identity server. The developer doesn't need to manually attach the access token since this method does it automatically.
```